lua/pac3/core/client/parts/proxy.lua

```diff
@@ -192,6 +192,30 @@ end
 
 PART.Inputs = {}
 
+PART.Inputs.property = function(self, property_name, field)

```

Comment:
I'm a bit concerned about the performance drop from this. Can you provide a test outfit where I can investigate what calls this functions?

Reply:
[property_proxy.txt](https://github.com/CapsAdmin/pac3/files/7646693/property_proxy.txt)